src/plugins/analysis/string_evaluation/test/test_string_eval.py

```diff
@@ -24,17 +24,19 @@ class TestStringEval(unittest.TestCase):
     def test_eval_strings(self):
         input_data = ['this string is useful', 'garbage n$%Schtz', '/an/interesting/directory']
         result = eval_strings(input_data)
-        self.assertEqual(
-            result, ['/an/interesting/directory', 'this string is useful', 'garbage n$%Schtz'], 'Fehler beim bewerten'
-        )
+        assert result == [
+            '/an/interesting/directory',
+            'this string is useful',
+            'garbage n$%Schtz',
+        ], 'Fehler beim bewerten'
```

Comment:
Err I don't think we should have german comments in our code.
I'll fix this in a future PR.